backends/nxp/tests/test_neutron_backend.py

```diff
@@ -53,3 +77,307 @@ def test_neutron_backend__linear_softmax_model__payload_header_formatless():
     assert payload[6] == 0x0  # Map 0-th Neutron output to 0-th model output
     assert all(byte == 0x0 for byte in payload[7:16])  # Aligned to 16 bytes
     assert payload[17] != 0x0  # Followed by non-zero content
+
+
+def test_lowered_program_and_tflite_output_match__conv2d__no_bias(mocker):
+    converter_spy = mocker.spy(EdgeProgramToIRConverter, "convert_program")
+
+    model = Conv2dModule(bias=False)
+    input_shape = (1, 4, 32, 32)
+
+    # Run conversion
+    to_quantized_edge_program(
+        model, input_shape, use_neutron_for_format_conversion=False
+    )
+
+    # Capture generated model
+    tflite_flatbuffers_model, io_formats = converter_spy.spy_return
+
+    tflite_model = Model.GetRootAs(tflite_flatbuffers_model)
+    sub_graph = tflite_model.Subgraphs(0)
+
+    assert sub_graph.OperatorsLength() == 1
+    assert sub_graph.Operators(0).BuiltinOptionsType() == BuiltinOptions.Conv2DOptions
+
+    # Capture converted program
+    exported_program: ExportedProgram = converter_spy.call_args.args[1]
+
+    input_data = (
+        (torch.randn(input_shape, dtype=torch.float32) * 50)
+        .type(torch.int8)
+        .detach()
+        .numpy()
+    )
+    input_data_tflite = np.transpose(input_data, [0, 2, 3, 1])
+
+    # Execute program and TFLite model
+    program_executor = EdgeProgramExecutor(exported_program)
+    tflite_executor = TFLiteExecutor(model_content=tflite_flatbuffers_model)
+
+    output_edge = program_executor.inference(input_data)
+    output_tflite = tflite_executor.inference(input_data_tflite)
+
+    output_tflite = np.transpose(output_tflite, [0, 3, 1, 2])
+
+    # Outputs difference is smaller than 1 (rounding error in quantization)
+    assert np.max(np.abs(output_edge - output_tflite)) <= 1
+
+
+def test_conv_fc__lowered_program_and_tflite_output_match(mocker):
```

Comment:
This test was moved to `test_neutron_backend_executor.py` by Jake, to fit the Meta internal test environment.
i.e. everything using the `executors` are in `test_neutron_backend_executor.py`

Remove from here.